docs/api/v1.24.md

```diff
@@ -2648,6 +2648,7 @@ Docker daemon report the following event:
 **Status codes**:
 
 -   **200** – no error
+-   **400** - bad parameter
```

Comment:
Actually I am OK with the change into `bad request`. While I am afraid that we should change all the api docs and swagger.yml to keep consistency. Is it OK for that? @thaJeztah